01-getting-started/projects/final/app/src/main/java/com/raywenderlich/podplay/ui/PodcastDetailsFragment.kt

```diff
@@ -0,0 +1,153 @@
+ *   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *   copies of the Software, and to permit persons to whom the Software is
+ *   furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *   Notwithstanding the foregoing, you may not use, copy, modify, merge, publish,
+ *   distribute, sublicense, create a derivative work, and/or sell copies of the
+ *   Software in any work that is designed, intended, or marketed for pedagogical or
+ *   instructional purposes related to programming, coding, application development,
+ *   or information technology.  Permission for such use, copying, modification,
+ *   merger, publication, distribution, sublicensing, creation of derivative works,
+ *   or sale is expressly withheld.
+ *
+ *   This project and source code may use libraries or frameworks that are
+ *   released under various Open-Source licenses. Use of those libraries and
+ *   frameworks are governed by their own individual licenses.
+ *
+ *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ */
+
+package com.raywenderlich.podplay.ui
+
+import android.content.Context
+import android.os.Bundle
+import android.text.method.ScrollingMovementMethod
+import android.view.*
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.raywenderlich.podplay.R
+import com.raywenderlich.podplay.adapter.EpisodeListAdapter
+import com.raywenderlich.podplay.databinding.FragmentPodcastDetailsBinding
+import com.raywenderlich.podplay.util.GlideApp
+import com.raywenderlich.podplay.viewmodel.PodcastViewModel
+
+class PodcastDetailsFragment : Fragment(), EpisodeListAdapter.EpisodeListAdapterListener {
+
+  private val podcastViewModel: PodcastViewModel by activityViewModels()
+  private lateinit var databinding: FragmentPodcastDetailsBinding
+  private lateinit var episodeListAdapter: EpisodeListAdapter
+  private var listener: OnPodcastDetailsListener? = null
+
+  companion object {
+    fun newInstance(): PodcastDetailsFragment {
+      return PodcastDetailsFragment()
+    }
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setHasOptionsMenu(true)
+  }
+
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
```

Comment:
formatting

Reply:
Will update with RW code formatting style.

Reply:
https://trello.com/c/gKEwBixK/25-apply-rw-style-format-throughout-the-project